src/components/Data/Layout.vue

```diff
@@ -44,6 +45,9 @@ export default {
   computed: {
     ...mapGetters('index', ['loadingIndexes', 'loadingCollections']),
     ...mapGetters('auth', ['isAuthenticated']),
+    paneSize() {
+      return `${getLocalStorageItem('paneSize')}px`;
+    },
```

Comment:
Since localStorage.get won't be reactive, I'm wondering if it is necessary to use a computed there ?

Reply:
👍 👍 👍 👍 👍 👍 Good catch!
Definitely right. `panelSize` should be part of the component state and get synchronized with Local Storage via component logic!

Reply:
I'm afraid that I don't understand clearly what you mean...can you explain to me again please ?

Reply:
A `computed` is usefull because the inner used properties are observed so the computed will be recalculated following it properties observers
Here `localStorage.getItem` is a call to local storage and cannot be observed so even if we use `localStorage.setItem`  the `computed` won't be updated
This is why IMO it is not so usefull to use a `computed` here
You could use a `data` to store localStorage value at `mounted` and then update it at every `localStorage.setItem`

Reply:
A little bit more of explanations:
* a value being _observed_ means it's part of a special VueJS object, encapsulating all the attributes in getters and setters implementing the Observable Pattern, enabling data binding into the application;
* unfortunately, this is pure VueJS internal "magic" and doesn't affect the standard API exposed by the browser (like the `localStorage` object);
* a `computed` value is a value in the scope of a VueJS component of which you define the behavior of the getter (and that has no setter): the value of the getter will be updated every time the source of this value changes (i.e. all the observable values used to compute the value), but since VueJS cannot observe `localStorage.get`, the value of the getter will not be updated if the value in the Local Storage changes.

Sounds clear? This is an advanced key concept and if you need more help, we can talk directly on Discord.

Reply:
Very clear explanations, thank you !